### DISCUSSION TITLE
How can i use blade-icons in Jigsaw?

### DISCUSSION THREAD

**Discussion by cristiangervasi** [tighten/jigsaw#647] [Q&A] [answered]
Hi,
I would like to use the blade-icons package in a Jigsaw project.
I tried installing the package with `composer require blade-ui-kit/blade-heroicons` to use the heroicons library but without success, the component cannot be located and therefore used.
Is there a way to use blade-icons and similar blade components packages in Jigsaw?
Thanks

**Comment by bakerkretzmar** [comment, marked as answer]
At the moment no, they don't work out of the box. You can almost definitely get the working if you look at the package's service provider and re-implement it in your Jigsaw site, in `bootstrap.php` or something. This is on our radar and we're going to try to make installing packages like this simpler in v2, I've added this particular package to my list to come back to 👍🏻

**Comment by neerajsohal** [comment]
@cristiangervasi were you able to get this working? I am having a real hard time with this
